src/core/attrsgenerator.cpp

```diff
@@ -41,12 +41,17 @@ AttrsGeneratorPtr AttrsGenerator::parse(const AttributesScope& attrsScope,
     int size = cmd.toInt(&sizeIsValid);
     if (sizeIsValid) { // cmd is just an interger
         _cmd = QString("*%1;min").arg(size);
+        cmds.push_back(cmd);
         cmds.push_back("min");
-    } else {
+    } else if (cmd.indexOf(";") != -1){
```

Comment:
As described in the [docs for this function](https://evoplex.org/api/0.2.1/classevoplex_1_1AttrsGenerator.html#af0482202bb90b305b8d2c0e2a5069693), the commands `*min` and `*max` are also valid. So, we should not assume that all commands have `;`.

Reply:
Hm I went back to the docs and I think it now covers all the cases. Please, let me know if this fixes it!

Reply:
I think the other cases were working fine already. The problem was with the `int` command only, and the line 44 should be enough to fix it. :+1:

Reply:
Hm, my build crashes whenever I input a invalid command (e.g. ``maxs``) that's why I included the extra conditional.